### PR TITLE
New 8086 instructions cheat sheet

### PR DESCRIPTION
**Pull request by jophab**
**What does your Instant Answer do?**
shows 8086 instructions and explanations
**What is the data source for your Instant Answer? (Provide a link if possible)**
http://users.utcluj.ro/~elupu/Curs/fileloader.php?fileName=upload/Cursuri/Univ.Nord_BM1/Curs_5/8086_instruction_set.pdf
**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**
It covers more topics
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
students
**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
no
**Are you having any problems? Do you need our help with anything?**
no
**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

3d

-----
IA Page: https://duck.co/ia/view/8086_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/cpp.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
+{
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/cpp.json]: @jophab This shouldn't be here. You should use different branches for different PRs

**Comment by codenirvana** [comment]
There's some indentation and unnecessary new lines problems.
Use tools like ```JSONLint``` or ```JSONFormatter``` to fix this problem for you!

**Comment by jophab** [comment]
@moollaza  @mintsoft  @codenirvana How is it now?

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com

**Comment by moollaza** [comment]
@jophab Thanks! This LGTM :+1:

**Pull request merged**